app/testOnly/views/TestOnlyStartPage.scala.html

```diff
@@ -745,9 +745,7 @@
         )),
         errorMessage = form.error("siaDebtTotalAmount").map(e => ErrorMessage.errorMessageWithDefaultStringsTranslated(
             content = Text(e.message)
-        )),
```

Comment:
isn't what you've removed here still relevant? Doesn't it get used to stick into the eligibility result in the stubs?

Reply:
it could be, it was one of the bits that jamie pointed out, but if it should be there I can add it back in